src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/_help.py

```diff
@@ -13,7 +13,7 @@
 
 helps['batch account'] = """
     type: group
-    short-summary: Manage your Batch accounts.
+    short-summary: Manage Azure Batch accounts.
```

Comment:
Group header descriptions should probably be branded correctly, so it would be `Azure Container` changed in the other location. The general documentation rules I tried to follow as far as branding were:

* No acronyms in group summaries
* Capitalization consistent with branding in group summaries
* Acronyms allowed if they are clearly defined or common (i.e. VM); Lowercase allowed if a common engineering term (i.e. "container").  Since "Batch" is not an unambiguous engineering term in this context it remains capitalized throughout to refer specifically to the Azure services concept.

This is consistent with the way that our (mostly informal) style guide has worked for web content, but I'm open to discussing it.

Reply:
Those seem like pretty thorough guides so I'm happy. :)